Extension/src/LanguageServer/client.ts

```diff
@@ -322,6 +323,15 @@ class DefaultClient implements Client {
             storagePath = path.join(storagePath, serverName);
         }
 
+         // Check for vcpkg instance.
+        if (existsSync(util.getVcpkgPathDescriptorFile())) {
```

Comment:
This belongs in common.ts (probably just merged in with getVcpkgRoot()).  We don't want to do this N times for N clients in a multi-root workspace.